WindowState/README.md

```diff
@@ -0,0 +1,127 @@
+# WindowState - Surface Duo Compose SDK
+
+**WindowState** is a component for Jetpack Compose, help you easily get details about the window state of the dual-screen, foldable and large screen devices, using the Google [Jetpack WindowManager](https://developer.android.com/jetpack/androidx/releases/window) library.
```

Comment:
Same small typo here - I think it should be:
 "WindowState is a component for Jetpack Compose that helps you easily get details about the window state of dual-screen, foldable, and large screen devices..."